themes/docsmith/layouts/partials/community_link.html

```diff
@@ -1,3 +1,3 @@
-<h3>Join our Community</h3>
+<h2 id="join-our-community">Join our Community</h2>
```

Comment:
When this used to be a H3, it would either:

- Appear as a subsection of the 'More Information' H2 section, if the guide had 'More Information' links in the frontmatter; or:
- Appear as a subsection of the last section in the guide

The first scenario was ok, but the second scenario was not good. Promoting 'Join our Community' to an H2 to avoid this.

Also adding an ID so it can be linked to from the nav.